xonsh/aliases.py

```diff
@@ -1,28 +1,27 @@
 # -*- coding: utf-8 -*-
 """Aliases for the xonsh shell."""
 
+from argparse import Action, ArgumentParser
+import builtins
+from collections.abc import MutableMapping, Iterable, Sequence
+from functools import lru_cache
```

Comment:
I think it is nicer to have them in length order, like before.  I guess this doesn't block the PR, but I'll probably change it back eventually :)